openreview/venue/matching.py

```diff
@@ -0,0 +1,999 @@
+                                }
+                            }
+                        },
+                        'aggregate_score_invitation': {
+                            'order': 9,
+                            'description': 'Invitation to store aggregated scores',
+                            'value': {
+                                'param': {
+                                    'type': 'string',
+                                    'regex': '{}/.*'.format(venue.id),
+                                    'default': self._get_edge_invitation_id('Aggregate_Score'),
+                                    'hidden': True
+                                }
+                            }
+                        },
+                        'conflicts_invitation': {
+                            'order': 10,
+                            'description': 'Invitation to store conflict scores',
+                            'value': {
+                                'param': {
+                                    'type': 'string',
+                                    'regex': '{}/.*'.format(venue.id),
+                                    'default': venue.get_conflict_score_id(self.match_group.id),
+                                }
+                            }
+                        },
+                        'assignment_invitation': {
+                            'order': 11,
+                            'description': 'Invitation to store paper user assignments',
+                            'value': {
+                                'param': {
+                                    'type': 'string',
+                                    'const': venue.get_paper_assignment_id(self.match_group.id),
+                                    'hidden': True
+                                }
+                            }
+                        },
+                        'deployed_assignment_invitation': {
+                            'order': 12,
+                            'description': 'Invitation to store deployed paper user assignments',
+                            'value': {
+                                'param': {
+                                    'type': 'string',
+                                    'const': venue.get_paper_assignment_id(self.match_group.id, deployed=True),
+                                    'hidden': True
+                                }
+                            }
+                        },
+                        'invite_assignment_invitation': {
+                            'order': 13,
+                            'description': 'Invitation used to invite external or emergency reviewers',
+                            'value': {
+                                'param': {
+                                    'type': 'string',
+                                    'const': venue.get_paper_assignment_id(self.match_group.id, deployed=True),
```

Comment:
Should this one instead be:
`venue.get_paper_assignment_id(self.match_group.id, invite=True)`?

Reply:
yes! Good catch, I'll make the change